app/helpers/routing_helper.rb

```diff
@@ -0,0 +1,54 @@
+require 'httparty'
+require 'json'
+
+module RoutingHelper
+  def self.seconds_to_minsec(sec)
+    format("%<minutes>.2d:%<seconds>.2d", minutes: sec / 60, seconds: sec % 60)
+  end
+
+  def self.valid_coordinates(coordinates)
+    return false if coordinates.blank?
+
+    regex = /^[0-9]{1,2}(\.[0-9]{1,6})?,[0-9]{1,2}(\.[0-9]{1,6})?$/
+    coordinates.match(regex)
+  end
+
+  def self.calculate_route(start, destination)
+    return nil unless valid_coordinates(start) && valid_coordinates(destination) # No route requested
```

Comment:
The `nil` is implicit:
```suggestion
    return unless valid_coordinates(start) && valid_coordinates(destination) # No route requested
```

We should probably not fail silently here, users may wonder why their request isn't working (Did I use the wrong parameter names? Did I format my coordinates inproperly?). Let's mark this as a todo or fixme before merging - we don't have a UI for displaying errors yet

Reply:
Maybe something like window.alert()?

Reply:
Hm, I don't think we necessarily need to _introduce_ error reporting just yet, only keep it in mind. It was not in the acceptance criteria and there simply isn't UI for that yet, window.alert sure isn't the end solution. I would be fine with just leaving a TODO comment and / or console log for now :)